packages/permissions/src/utils/has-permissions.ts

```diff
@@ -141,3 +178,98 @@ export const getInvalidPermissions = (
     isNil(actualPermissions[toCanCase(demandedPermission)])
   );
 };
+
+const hasDemandedDataFenceForStore = (
+  authorizationContext: TAuthorizationContext,
+  options: {
+    actualDataFence: TActualDataFence;
+    demandedDataFence: TDemandedDataFence;
+  }
+): boolean => {
+  const hasDemandedPermission = hasPermission(options.demandedDataFence.name, {
+    [options.actualDataFence.name]: true,
+  });
+  if (!hasDemandedPermission) return false;
+  switch (options.demandedDataFence.group) {
+    case DATA_FENCE_GROUPS.ORDERS: {
```

Comment:
Why does it need to be hardcoded? 🤔

Reply:
I don't get it. what's hardcoded?
we need the `group` to determine how to match the resource value to the given datafence